### PR TITLE
Added Audioop to Requirements

### PR DESCRIPTION
**Pull request by Zachdehooge**
Added Audioop to Requirements as this was something it yelled at me about when building the project in main

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 discord.py==2.4.0
 python-dotenv==1.0.1
 beautifulsoup4==4.12.3
-requests==2.32.3
\ No newline at end of file
+requests==2.32.3
+audioop-lts==0.2.1
```
Comment (Arch881010) [inline requirements.txt:5]: All looks good to me except for here. 
image
audioop is being emoved in py 3.13
Are you sure it's not accidentally picking up your other projects?

Comment (Zachdehooge) [reply requirements.txt:5]: It might, let me check because I think it had a different version selected in VSCode

Comment (Zachdehooge) [reply requirements.txt:5]: That was it, closing without merging

**Comment by Zachdehooge** [comment]
Different version of Python selected in VSCode. Resolved issue client side

**Pull request closed without merging**